certify-service/src/main/java/io/mosip/certify/enums/CredentialStatusPurpose.java

```diff
@@ -0,0 +1,28 @@
+package io.mosip.certify.enums;
+
+import io.mosip.certify.core.exception.CertifyException;
+
+import java.util.Arrays;
+
+public enum CredentialStatusPurpose {
+    REVOCATION("revocation"),
+    SUSPENSION("suspension");
```

Comment:
There can be more purposes, can you keep the support list as a property file config ?